### DISCUSSION TITLE
🚀 Feature request: in a Grid field, make it possible to have the handle/trash icons as the first column rather than the last column

### DISCUSSION THREAD

**Discussion by strudelmedia** [ExpressionEngine/ExpressionEngine#4094] [Feature Requests]
### Suggest an idea for ExpressionEngine

**Is your feature request related to a problem? Please describe.**
Currently, with a Grid field with several columns, I have to scroll right quite a bit to access the handle to reorder the rows. This becomes an issue particularly when using the Grid for images: if the first column is the image, and that is followed by several other columns (photo credit, caption, etc.) then to reorder I can't see the images any more, which means I can't reorder things visually.

**Describe the solution you'd like**
I would like the default position of the handle/trash icon to be the first column, or at least make it possible to change it from the last column to the first.

**Describe alternatives you've considered**
I think this is the easiest solution. Also, from a UX point of view, it's not good to hide an important control (having it only be visible when scrolled all the way to the right).

**Teachability, Documentation, Adoption, Migration Strategy**
I think the best would be to change the default position of these icons.

Unfortunately, I don't know nuttin' about how all this works so I wouldn't be able to contribute to the implementation.

- [ ] I am capable and would like to work on implementation of this feature if it is considered.

**Comment by intoeetive** [comment]
@Yulyaswan fyi

**Comment by Yulyaswan** [comment]
PR - https://github.com/ExpressionEngine/ExpressionEngine/pull/5203
PR - #5209 
PR - #5210
